src/status_checker.c

```diff
@@ -241,11 +238,52 @@ const u8 gDungeonCamouflageTypes[76] = {
 
 extern s16 gUnknown_80F4DC6;
 extern u8 *gUnknown_80FEFF4[];
-extern void sub_80763F8(struct DungeonEntity *, struct DungeonEntity *, u32);
+
+extern u32 gUnknown_80F51C4[];
+
+bool8 sub_805AFA4(struct DungeonEntity * pokemon, struct DungeonEntity * target, struct PokemonMove *move, u32 param_4)
```

Comment:
This file was originally for helper functions for the Status Checker IQ, but there's now a lot more functions. Either the file should be renamed or the other functions should be split off to a new file.

Reply:
I'm tempted to just rename the file since there isn't any other Status Checker IQ funcs aside from those 2. Might just do the `code_` for now until we get more of these moveActions/StatusTarget funcs decomped and we can then group them better.

Reply:
I see a clear split between `PierceOrbAction()` and `CanUseOnSelfWithStatusChecker()`. Anything before that deals with applying statuses, and anything after that deals with various conditions for being able to use a move. There's a sufficient amount of labeled code to come up with names like `status_actions.c` and `move_checks.c` rather than nondescript `code_` names.

Reply:
Split into `status_actions.c` and `move_checks.c` as requested